### DISCUSSION TITLE
excellent

### DISCUSSION THREAD

**Discussion by adel-lamloum** [flxzt/rnote#910] [General]
البرنامج ممتاز جدا واحببت ان اشكرك باللغة العربيه لكي تعلم ان محبين برنامجك من كل انحاء العالم  اتمني ان تستمر

**Comment by flxzt** [comment]
Thanks for the kind words :)

Just for other readers - this is what is says (per google translate): "The program is very excellent and I wanted to thank you in Arabic so that you know that your program is loved from all over the world. I hope you continue."

**Comment by mahdihasan2006** [comment]
এবং আমি বাংলাদেশ থেকে বলছি।

And I am from Bangladesh.
